packages/metal-soy/test/Soy.js

```diff
@@ -1,5 +1,6 @@
 'use strict';
 
+import { core } from 'metal';
```

Comment:
The tests are there because I wasn't sure if "DOM" related test should have been placed in the `metal` package, and `metal-soy` had everything needed. 

It might be a better idea to create a new `metal-custom-element` package and add all the related code in there.